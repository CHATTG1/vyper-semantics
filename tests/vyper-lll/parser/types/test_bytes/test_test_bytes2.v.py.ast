%pgm(, ,
  %fdecl(%@public, foo, %param(x, %bytesT(100)), %bytesT(100),
    %assign(%var(y), %var(x))
    %return(%var(y)))
)
