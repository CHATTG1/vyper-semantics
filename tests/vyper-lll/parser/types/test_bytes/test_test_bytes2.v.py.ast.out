$%seq ( $return ( 0 , $lll ( $%seq ( $%seq ( $mstore ( 28 , $calldataload ( 0 ) ) , $mstore ( 32 , 1461501637330902918203684832716283019655932542976 ) , $mstore ( 64 , 170141183460469231731687303715884105727 ) , $mstore ( 96 , -170141183460469231731687303715884105728 ) , $mstore ( 128 , 1701411834604692317316873037158841057270000000000 ) , $mstore ( 160 , -1701411834604692317316873037158841057280000000000 ) ) , $if ( $eq ( $mload ( 0 ) , 818467290 ) , $%seq ( $calldatacopy ( 320 , 4 , 32 ) , $assert ( $iszero ( $callvalue ) ) , $%seq ( $calldatacopy ( 352 , $add ( 4 , $calldataload ( $add ( 4 , 0 ) ) ) , 132 ) , $assert ( $le ( $calldataload ( $add ( 4 , $calldataload ( $add ( 4 , 0 ) ) ) ) , 100 ) ) ) , $with ( $_source , 352 , $with ( $_sz , $add ( 32 , $mload ( $_source ) ) , $assert ( $call ( $add ( 18 , $div ( $_sz , 10 ) ) , 4 , 0 , $_source , $_sz , 544 , $_sz ) ) ) ) , $with ( $_loc , 544 , $%seq ( $mstore ( $sub ( $_loc , 32 ) , 32 ) , $return ( $sub ( $_loc , 32 ) , $ceil32 ( $add ( $mload ( $_loc ) , 64 ) ) ) ) ) , $stop ) ) ) , 0 ) ) )
